File: php/manage-navigation/GetArticlesSlug.php
<?php
// DocSection: managing_navigation_articles_slugs
// Tip: Find more about PHP SDKs at https://kontent.ai/learn/php
// Defined by Composer to include required libraries
require __DIR__ . '/vendor/autoload.php';

use Kontent\Ai\Delivery\DeliveryClient;

$client = new DeliveryClient('8d20758c-d74c-4f59-ae04-ee928c0816b7');

// Gets navigation items and their linked items
$root = $client->getItem('root_navigation_item', (new QueryParams())
            ->depth(5));

// Gets specific elements of all articles
$articles = $client->getItems((new QueryParams())
            ->equals('system.type', 'article')
            ->elements(array("title", "url")));
// EndDocSection
?>

File: java/manage-navigation/GetArticlesSlug.java
// DocSection: managing_navigation_articles_slugs
// Tip: Find more about Java SDK at https://kontent.ai/learn/java
import kentico.kontent.delivery.*;

// Initializes a DeliveryClient
DeliveryClient client = new DeliveryClient("<YOUR_PROJECT_ID>");

// Registers the model class for navigation items
// Tip: Create strongly typed models according to https://kontent.ai/learn/strongly-typed-models
client.registerType(NavigationItem.class);

// Gets navigation items and their linked items
CompletionStage<NavigationItem> root = client.getItem(
    "root_navigation_item",
     NavigationItem.class,
     DeliveryParameterBuilder.params()
        .linkedItemsDepth(5)
        .build()

// Registers the model class for articles
client.registerType(Article.class);

// Gets specific elements of all articles
CompletionStage<List<Article>> articles = client.getItems(
    Article.class, 
    DeliveryParameterBuilder.params()
        .projection("title", "url")
        .build()
);
// EndDocSection